curtsies/formatstring.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import unicode_literals
```

Comment:
After reading this, let's ignore this whole file. I think Black is good going forward, but some of this stuff gets very hard to read since it wasn't written for it.